pkg/services/auth/identity/requester.go

```diff
@@ -13,6 +13,8 @@ type Requester interface {
 	// GetNamespacedID returns the namespace and ID of the active entity.
 	// The namespace is one of the constants defined in pkg/services/auth/identity.
 	GetNamespacedID() (namespace string, identifier string)
+	// GetUserUID returns the user uid for the entity
+	GetUserUID() string
```

Comment:
Added this to support getting the uid for the signed in user

Reply:
Could we rename this to GetUID?

Identity can be more than a user, e.g. service account or api key. Right now UID should be set for users and service accounts only. We could add it to api key but they are deprecated to I don't see the point.

Reply:
We could do that, as long as we're comfortable that the UID will uniquely identify an identity and we don't need to also know the type. I considered simply duplicating GetID and GetNamespacedID and using the same namespace syntax which would get around that issue. Happy to implement that if you think it makes more sense.